src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -111,11 +112,18 @@ public async Task<AmlResult> PerformAmlCheckAsync(string appId, AsymmetricCipher
             if (string.IsNullOrEmpty(authDigest))
                 throw new InvalidOperationException("Could not sign request");
 
+            string SDKversion = typeof(YotiClientEngine).GetTypeInfo()?.Assembly?.GetName()?.Version?.ToString();
+            if (SDKversion == null)
+            {
+                SDKversion = "Unknown";
```

Comment:
Will it be useful to send version value as `Unknown`.
If there is an error, I would rather log it and not send the value as `Unknown`, your call.

Reply:
If we're logging an error, it's not likely we'll be able to see it. I guess it wouldn't harm to also log it though, I'll add it. 

If we send the header as "Unknown", then we will be able to see it in the logs.

Reply:
I guess you could see it when you run your test cases locally and will be helpful.
It's your call.